app/views/dashboard/_user_topics.html.erb

```diff
@@ -0,0 +1,60 @@
+<div class="card-view" style="margin-left: -13px;">
+  <% current_user.subscriptions(:tag).each do |tag| %>
+    <div class="card shadow-sm p-2 mb-3 bg-white rounded" >
+      <div class="card-header mt-2" style=" padding:0.8em; background-color: inherit; border:none;">
+        <h4 style="display: inline-block;"><%= tag.tagname %></h4>
+        <p style="display: inline-block; color: #808080;"><a style="text-decoration: underline; color: #808080;" href="/tag/<%= tag.tagname %>"><%= Tag.follower_count(tag.tagname) %> people</a> discussing</p>
+      </div>
+      <div class="card-body" style="padding:0.8em;">
+        <div class="node-list">
+          <% Tag.find_nodes_by_type(tag.tagname, type = 'note', limit = 3).each do |node|  %>
+            <div class="node-body">
+              <% if node.main_image %>
+                <img class="rounded-circle" id="profile-photo" style="width:20px; height:20px; margin-right:8px; display: inline-block;" src="<%= node.main_image.path(:default) %>" />
+              <% elsif node.scraped_image %>
+                <img class="rounded-circle" id="profile-photo" style="width:20px; height:20px; margin-right:8px; display: inline-block;" src="<%= node.scraped_image %>" />
+              <% else %>
+                <div class="circle"></div>
+              <% end %>
+              <p style="display: inline-block; margin-left: 10px;"><a style="color: inherit;" <% if @widget %>target="_blank"<% end %> href="<%= node.path %>"><%= (node.type == 'note') ? node.title : node.latest.title %></a></p>
+              <p style="display: inline-block; margin-left: 10px;color: #808080;">by <a style="color: #808080;" <% if @widget %>target="_blank"<% end %> href="/profile/<%= node.author.name %>">@<%= node.author.name %></a></p>
+            </div>
+          <% end %>
+        </div>
+      </div>
+      <div class="card-footer" style="background-color: inherit; border:none;">
+        <a style="text-decoration: underline; color: #808080; display: inline-block;" href="/tag/<%= tag.tagname %>"><%= Tag.follower_count(tag.tagname).to_i-1 %> <%= t('tag.index.more_posts') %> <i class="fa fa-angle-double-right"></i></a>
```

Comment:
Similar as above, Not just notes, but need to include comments,questions too.
This will be added in profile page soon too.

Reply:
Questions and notes have already been added!

Reply:
Here notes are covering all the posts made. I think the only portion not covered here are the wikis and the comments. We can discuss about them with @jywarren !